portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +24,18 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h3>Hello Andrew!</h3>");
+    String[] ArrayList = {"Hey!", "Check", "here!"};
+    String To_Json = convertToJson_Gson(ArrayList);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(To_Json);
+  }
+  private String convertToJson_Gson(String [] ArrayList) {
+    Gson gson = new Gson();
+    String ToJson = gson.toJson(ArrayList);
+    return ToJson;
```

Comment:
Nit: usually if a local variable is not used elsewhere except in a return statement, it'd be good to avoid it in the first place so that readers don't need to remember one more variable. Just do:

return gson.toJson(arrayList);